sql-files/main.sql

```diff
@@ -916,3 +918,53 @@ CREATE TABLE IF NOT EXISTS `storage` (
   PRIMARY KEY (`id`),
   KEY `account_id` (`account_id`)
 ) ENGINE=MyISAM;
+
+CREATE TABLE IF NOT EXISTS `rodex_items` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`mail_id` BIGINT(20) NOT NULL DEFAULT '0',
+	`nameid` INT(11) NOT NULL DEFAULT '0',
+	`amount` INT(11) NOT NULL DEFAULT '0',
+	`equip` INT(11) UNSIGNED NOT NULL DEFAULT '0',
+	`identify` SMALLINT(6) NOT NULL DEFAULT '0',
+	`refine` TINYINT(3) UNSIGNED NOT NULL DEFAULT '0',
+	`attribute` TINYINT(4) NOT NULL DEFAULT '0',
+	`card0` SMALLINT(11) NOT NULL DEFAULT '0',
+	`card1` SMALLINT(11) NOT NULL DEFAULT '0',
+	`card2` SMALLINT(11) NOT NULL DEFAULT '0',
+	`card3` SMALLINT(11) NOT NULL DEFAULT '0',
+  `opt_idx0` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val0` SMALLINT(5) NOT NULL DEFAULT '0',
+  `opt_idx1` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val1` SMALLINT(5) NOT NULL DEFAULT '0',
+  `opt_idx2` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val2` SMALLINT(5) NOT NULL DEFAULT '0',
+  `opt_idx3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val3` SMALLINT(5) NOT NULL DEFAULT '0',
+  `opt_idx4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val4` SMALLINT(5) NOT NULL DEFAULT '0',
+	`expire_time` INT(11) UNSIGNED NOT NULL DEFAULT '0',
+	`bound` TINYINT(1) UNSIGNED NOT NULL DEFAULT '0',
+	`unique_id` BIGINT(20) UNSIGNED NOT NULL DEFAULT '0',
+	PRIMARY KEY (`id`),
+  KEY `mail_id` (`mail_id`)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS `rodex_mail` (
+	`id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+	`sender_name` VARCHAR(30) NOT NULL COLLATE 'utf8_unicode_ci',
+	`sender_id` INT(11) NOT NULL,
+	`receiver_name` VARCHAR(30) NOT NULL COLLATE 'utf8_unicode_ci',
+	`receiver_id` INT(11) NOT NULL,
+  `receiver_accountid` INT(11) NOT NULL,
+	`title` VARCHAR(50) NOT NULL COLLATE 'utf8_unicode_ci',
```

Comment:
I think the character set shouldn't be a column property, but rather a database (or at most, table) property, unless the client requires this to be utf8_unicode_ci (which I'm not aware of)